schemas/test/en/test_new_confirmation_question.json

```diff
@@ -0,0 +1,175 @@
+                                            "transforms": [
+                                                {
+                                                    "transform": "first_non_empty_item",
+                                                    "arguments": {
+                                                        "items": [
+                                                            {
+                                                                "source": "metadata",
+                                                                "identifier": "trad_as"
+                                                            },
+                                                            {
+                                                                "source": "metadata",
+                                                                "identifier": "ru_name"
+                                                            }
+                                                        ]
+                                                    }
+                                                }
+                                            ]
+                                        }
+                                    ]
+                                },
+                                "type": "General"
+                            },
+                            "type": "Question"
+                        },
+                        {
+                            "type": "ConfirmationQuestion",
+                            "id": "confirm-zero-employees-block",
+                            "question": {
+                                "type": "General",
+                                "answers": [
+                                    {
+                                        "type": "Radio",
+                                        "id": "confirm-zero-employees-answer",
+                                        "options": [
+                                            {
+                                                "label": "Yes",
+                                                "value": "Yes"
+                                            },
+                                            {
+                                                "label": "No",
+                                                "value": "No"
+                                            }
+                                        ],
+                                        "mandatory": true
+                                    }
+                                ],
+                                "id": "confirm-zero-employees-question",
+                                "title": "Did you answer the last question correctly?"
+                            },
+                            "routing_rules": [
+                                {
+                                    "block": "number-of-employees-total-block",
+                                    "when": {
```

Comment:
You could argue we need a single rule in here as that is how it would normally be used. Naturally this exercise the recursive element